### ISSUE TITLE
C++ MSVC compilation has many warnings

### ISSUE THREAD

**Issue by tbscope** [slint-ui/slint#9699]
### Bug Description

When compiling C++ code with MSVC I get the following warning message:

`..\slint-build\generated_include\slint_internal.h(404): warning C4624: 'slint::cbindgen_private::AccessibilityAction': destructor was implicitly defined as deleted`

I did not try to compile with another compiler to see if they generate a similar warning.

### Reproducible Code (if applicable)

```slint

```

### Environment Details

- Slint Version: 1.13.1
- Platform/OS: Windows 11
- Programming Language: C++ using MSVC version 14.44.35207
- Backend/Renderer: Skia

### Product Impact

No impact, not critical.

**Comment by ogoffart** [comment]
Is there more warning or just this one?
We do see this warning on our CI:
https://github.com/slint-ui/slint/actions/runs/18415570060/job/52478334297#step:9:818
But there is many warning with MSVC and we don't block our CI on it because nobody from the core dev team is using Windows as their workstation.

We do have CI checks for warnings in Gcc and Clang, but not on MSVC
https://github.com/slint-ui/slint/blob/7bb333c77dd477f8625c814be7f5b464bfd6553d/tests/driver/cpp/cppdriver.rs#L134

It would be worthwhile to fix these warnings.

**Comment by tbscope** [comment]
This is the only warning.
